wagtail/contrib/table_block/templates/table_block/widgets/table.html

```diff
@@ -19,6 +19,16 @@
     </div>
 </div>
 <br/>
+<div class="field">
+    <label for="{{ widget.attrs.id }}-handsontable-col-caption">{% trans 'Table caption' %}</label>
+    <div class="field-content">
+        <div class="input">
+          <input type="text" id="{{ widget.attrs.id }}-handsontable-col-caption" name="handsontable-col-caption"/>
+        </div>
+        <p class="help">{% trans 'Display caption - helpful for users with screen reader' %}</p>
```

Comment:
I suggest the following help text:

```suggestion
        <p class="help">{% trans 'A heading that identifies the overall topic of the table, and is useful for screen reader users.' %}</p>
```
The label of the field has "caption" in it, so, it may be redundant to repeat it in the help text. And I think that "useful" describes the function of the caption, somewhere between "helpful" and "important".

Reply:
👍 I like yours better than mine!